src/with_alloc/alloc_ringbuffer.rs

```diff
@@ -304,7 +238,7 @@ impl<T, SIZE: RingbufferSize> RingBufferWrite<T> for AllocRingBuffer<T, SIZE> {
             self.readptr += 1;
         }
 
-        let index = SIZE::mask(self.capacity, self.writeptr);
+        let index = mask(self.size, self.writeptr);
```

Comment:
Any occurrence of `SIZE::mask` is changed to this. Note the change some `self.capacity` to `self.size`.